src/compiler/types.ts

```diff
@@ -3770,30 +3770,31 @@ namespace ts {
 
     /* @internal */
     export interface SymbolLinks {
-        immediateTarget?: Symbol;           // Immediate target of an alias. May be another alias. Do not access directly, use `checker.getImmediateAliasedSymbol` instead.
-        target?: Symbol;                    // Resolved (non-alias) target of an alias
-        type?: Type;                        // Type of value symbol
-        uniqueESSymbolType?: Type;          // UniqueESSymbol type for a symbol
-        declaredType?: Type;                // Type of class, interface, enum, type alias, or type parameter
-        resolvedJSDocType?: Type;           // Resolved type of a JSDoc type reference
-        typeParameters?: TypeParameter[];   // Type parameters of type alias (undefined if non-generic)
-        outerTypeParameters?: TypeParameter[];  // Outer type parameters of anonymous object type
-        instantiations?: Map<Type>;         // Instantiations of generic type alias (undefined if non-generic)
-        mapper?: TypeMapper;                // Type mapper for instantiation alias
-        referenced?: boolean;               // True if alias symbol has been referenced as a value
+        immediateTarget?: Symbol;                 // Immediate target of an alias. May be another alias. Do not access directly, use `checker.getImmediateAliasedSymbol` instead.
+        target?: Symbol;                          // Resolved (non-alias) target of an alias
+        type?: Type;                              // Type of value symbol
+        uniqueESSymbolType?: Type;                // UniqueESSymbol type for a symbol
+        declaredType?: Type;                      // Type of class, interface, enum, type alias, or type parameter
+        resolvedJSDocType?: Type;                 // Resolved type of a JSDoc type reference
+        typeParameters?: TypeParameter[];         // Type parameters of type alias (undefined if non-generic)
+        outerTypeParameters?: TypeParameter[];    // Outer type parameters of anonymous object type
+        instantiations?: Map<Type>;               // Instantiations of generic type alias (undefined if non-generic)
+        inferredClassSymbol?: TransientSymbol;    // Symbol of an inferred ES5 constructor function

```

Comment:
ignore whitespace to highlight the actual change
(I added some whitespace on the other lines)